resources/probe.json.schema

```diff
@@ -26,10 +26,10 @@
           "annotations": {
             "type": "object",
             "properties": {
-              "name": { "type": "string" },
+              "model_name": { "type": "string" },
```

Comment:
I wasn't sure about this name change. Both "name" and "model_name" are present as metadata, and neither required by the probe object. Should we make it required here?

Reply:
Do we want the json_schema to map perfectly onto the Probe object, or might it make sense of have them hav slightly different requirements?